src/host/dns.h

```diff
@@ -23,21 +23,23 @@ namespace asynchost
       uv_getaddrinfo_cb cb,
       bool async)
     {
-      struct addrinfo hints;
+      struct addrinfo hints
+      {};
```

Comment:
```suggestion
      addrinfo hints{};
```

I think this is needed in C, but weird style for C++? It's certainly confused our formatter, which thinks this is a new struct type definition rather than a variable declaration + initialisation.